pkg/sql/delegate/show_changefeed_jobs.go

```diff
@@ -41,6 +41,7 @@ SELECT
   finished,
   modified,
   high_water_timestamp,
+  hlc_to_timestamp(high_water_timestamp) as readable_high_water_timestamp,
```

Comment:
@rohan-joshi do you have an opinion on this name?

Reply:
fwiw I prefer `high_water_timestamptz` which describes what is being shown (a timestamptz type), but Miles is ultimate arbiter here
<!-- Reviewable comment -OCJrV_U3EvTxXn4KidT:blp3chk -->
<!-- Sent from Reviewable.io -->